modules/snmp-dest/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+module_switch(ENABLE_SNMP_DEST "Enable snmp destination driver" NETSNMP_FOUND)
+
+set(snmpdest_SOURCES
+    "snmpdest-parser.c"
+    "snmpdest.c"
+    "snmpdest-parser.c"
+    "snmpdest-grammar.y"
+    "${CMAKE_CURRENT_BINARY_DIR}/snmpdest-grammar.c"
+)
+
+generate_y_from_ym(modules/snmp-dest/snmpdest-grammar)
+
+bison_target(snmpdest_GRAMMAR
+    ${CMAKE_CURRENT_BINARY_DIR}/snmpdest-grammar.y
+    ${CMAKE_CURRENT_BINARY_DIR}/snmpdest-grammar.c
+    COMPILE_FLAGS ${BISON_FLAGS})
+
+add_library(snmpdest SHARED ${snmpdest_SOURCES})
+
+target_include_directories(snmpdest
+  PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}
+  PRIVATE ${CMAKE_CURRENT_BINARY_DIR}
+)
+
+target_compile_definitions(snmpdest PUBLIC NETSNMP_CFLAGS)
+
+target_link_libraries(snmpdest PUBLIC ${NETSNMP_LIBS})
+target_link_libraries(snmpdest PUBLIC syslog-ng)
+
+install(TARGETS snmpdest LIBRARY DESTINATION lib/syslog-ng/ COMPONENT snmpdest)
```

Comment:
`add_test_subdirectory(tests)` is missing.

Reply:
added